app/components/Home.tsx

```diff
@@ -21,8 +23,14 @@ export default function Home(props: {
   // HOOKS
   const { data } = useTina(props);
 
+  const [width, setWidth] = useState(0);
+
+  useEffect(() => {
+    setWidth(typeof window !== "undefined" ? window.innerWidth : 0);
```

Comment:
Yes, this might be overkill, but it is logic that worked for me on a previous project. I might change it in the future to move away from using window.innerWidth however.